kong/plugins/http-log/schema.lua

```diff
@@ -42,6 +42,43 @@ return {
           }},
           { queue = typedefs.queue },
           { custom_fields_by_lua = typedefs.lua_code },
+          { enable_logged_headers_allow_list = {
```

Comment:
Either rename `enable_logged_headers_allow_list` to `enable_logged_header_allow_list` or vice versa in the specs.

Reply:
Ah yep, I see the typos now. Should be headers with 's' across the whole PR. Will address, thanks.

Reply:
@vm-001, this should be sorted now.